src/util/dump.ts

```diff
@@ -58,8 +58,7 @@ export async function getAndProcessStylesheets(downloader: Downloader, redisStor
               url = getFullUrl(url, cssUrl)
               url = url.indexOf('%') < 0 ? encodeURI(url) : url
```

Comment:
Really wonder about that line: it should be either the opposite… or we shoukd encode all the time.

Reply:
This row encodes the URL if it is not encoded. It does not allow encoding already encoded URL.